### PR TITLE
Do not check status codes from cucumber features

### PR DESCRIPTION
**Pull request by Martouta**
Checking status codes from cucumber does not work from `Chrome::Driver`, which is the browser used when we have the tag `@javascript`. This way it works anyway.
And either way it is wrong to test status codes from HTTP requests from Cucumber. Cucumber should do only UI related actions and assert from the UI.

### CODE REVIEW

File: test/integration/developer_portal/forums/admin/categories_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class DeveloperPortal::Forums::Admin::CategoriesControllerTest < ActionDispatch::IntegrationTest
```
Comment (Martouta) [inline test/integration/developer_portal/forums/admin/categories_controller_test.rb]: The tests that it is replacing are:
https://github.com/3scale/porta/blob/58f7353af17f210e2816c81eacc9ecf644d0a2eb/features/old/forums/categories.feature#L77-L106

For these steps:
https://github.com/3scale/porta/blob/58f7353af17f210e2816c81eacc9ecf644d0a2eb/features/step_definitions/forum/category_steps.rb#L43-L53

These are the routes we have for `forum/categories`:
image

So those in `admin` are for the admin portal, not the developer portal, and the test was checking that a logged buyer could not access it (404).

This was introduced in [this commit](https://github.com/3scale/system/commit/b9b09d9918291392459d7eb6707a61d49d0252cb).

In my opinion we shouldn't even have this test at all. But better safe than sorry so I rather asking you guys 😄

Comment (hallelujah) [reply test/integration/developer_portal/forums/admin/categories_controller_test.rb]: Side note:

Forum is deprecated feature. We should find a way to remove them :)

@3scale/product @secretsurf  @andreumasferrer what is the status of the forum in SaaS?

Comment (andreumasferrer) [reply test/integration/developer_portal/forums/admin/categories_controller_test.rb]: Forum feature is not officially deprecated in SaaS. The link is not included by default but the feature itself is still available. We would like to replace it by an integration with an external solution but we have no concrete plans yet.

Comment (Martouta) [reply test/integration/developer_portal/forums/admin/categories_controller_test.rb]: It doesn't matter for this anyway 😄

**Pull request merged**